src/libraries/Common/src/Interop/Windows/Advapi32/Interop.RegEnumKeyEx.cs

```diff
@@ -13,10 +13,10 @@ internal static partial class Interop
     internal static partial class Advapi32
     {
         [LibraryImport(Libraries.Advapi32, EntryPoint = "RegEnumKeyExW", StringMarshalling = StringMarshalling.Utf16)]
-        internal static partial int RegEnumKeyEx(
+        internal static unsafe partial int RegEnumKeyEx(
             SafeRegistryHandle hKey,
             int dwIndex,
-            [Out] char[] lpName,
+            ref char lpName,
```

Comment:
For my own education, why did you change this from a char array to a ref char?

Reply:
Because the name is in a `stackalloc`ed span (could make it a `char*`, don't remember why I didn't).